src/config.ts

```diff
@@ -195,6 +201,28 @@ export function checkMinimalConfigVersion(): void {
   }
 }
 
+export function isAfterEpoch(): boolean {
+  const config = getConfiguration();
+  const minVersionRaw = config.minVersion;
+  if (!minVersionRaw) {
+    return false;
+  }
+
+  const minVersion = parseVersion(minVersionRaw);
+  if (!minVersion) {
+    throw new Error(`Cannot parse the minimal version: "${minVersionRaw}"`);
+  }
+
+  const epochVersion = parseVersion(DEFAULTS_EPOCH_VERSION);
```

Comment:
The `epochVersion` should probably be passed in, once you have multiple epochs. Let's say we change the default for the preReleaseCommand in one of the next versions.

Reply:
Let's revisit that when the need arises, wdyt?

Reply:
Sure, that works for me 👍